universal-application-tool-0.0.1/test/services/cloud/azure/BlobStorageTest.java

```diff
@@ -51,14 +52,14 @@ public void getBlobUrl() {
 
   @Test
   public void getSasToken() {
-    String sasToken = blobStorage.getClient().getSasToken(TEST_FILE_NAME);
+    String sasToken = blobStorage.getClient().getSasToken(TEST_FILE_NAME, Optional.empty());
```

Comment:
do we need/want a test with this not optional?

Reply:
Added a test